discord-bot/src/server.ts

```diff
@@ -0,0 +1,16 @@
+import * as http from 'http'
+import * as process from 'process'
+
+const hostname = '0.0.0.0'
+const port = process.env.PORT ?? '8080'
+
+const server = http.createServer((req, res) => {
+  res.statusCode = 200
+  res.setHeader('Content-Type', 'text/plain')
+  res.end('Hello World')
```

Comment:
There should probably be some comment in here (or a response) explaining that this server just exists because Cloud Run things need to host a webserver so that Cloud Run can see that they are alive when it spins them up.